lib/web/fetch/webidl.js

```diff
@@ -216,6 +216,21 @@ webidl.util.IntegerPart = function (n) {
   return r
 }
 
+webidl.util.Stringify = function (V) {
+  const type = webidl.util.Type(V)
+
+  switch (type) {
+    case 'Symbol':
+      return `Symbol(${V.description})`
+    case 'Object':
+      return JSON.stringify(V)
```

Comment:
Actually... should be checking if it is a plain object and then use JSON.stringify? What if we have circulation? It would error.

Reply:
[object Object]